### ISSUE TITLE
IMI failure

### ISSUE THREAD

**Issue by waghanza** [the-benchmarker/web-frameworks#4838]
Hi @Yurunsoft,

With the last version (**2.0.22**), I have 
```php
@Yurunsoft 

```php
Fatal error: Uncaught Error: Class "ReturnTypeWillChange" not found in /usr/src/app/vendor/imiphp/imi/src/Bean/AnnotationParser.php:786
Stack trace:
#0 /usr/src/app/vendor/imiphp/imi/src/Bean/AnnotationParser.php(214): Imi\Bean\AnnotationParser->getPHPMethodAnnotations(Object(ReflectionMethod))
#1 /usr/src/app/vendor/imiphp/imi/src/Bean/AnnotationParser.php(202): Imi\Bean\AnnotationParser->parseMethod(Object(ReflectionClass), Object(ReflectionMethod))
#2 /usr/src/app/vendor/imiphp/imi/src/Bean/AnnotationParser.php(79): Imi\Bean\AnnotationParser->parseMethods(Object(ReflectionClass))
#3 /usr/src/app/vendor/imiphp/imi/src/Bean/AnnotationParser.php(153): Imi\Bean\AnnotationParser->parse('Imi\\Aop\\Annotat...')
#4 /usr/src/app/vendor/imiphp/imi/src/Bean/AnnotationParser.php(76): Imi\Bean\AnnotationParser->parseClass(Object(ReflectionClass))
#5 /usr/src/app/vendor/imiphp/imi/src/Bean/Annotation.php(144): Imi\Bean\AnnotationParser->parse('Imi\\Config\\Anno...', true, '/usr/src/app/ve...')
#6 /usr/src/app/vendor/imiphp/imi/src/Bean/AnnotationLoader.php(93): Imi\Bean\Annotation->Imi\Bean\{closure}('Imi\\Config\\Anno...', '/usr/src/app/ve...')
#7 /usr/src/app/vendor/imiphp/imi/src/Bean/AnnotationLoader.php(68): Imi\Bean\AnnotationLoader->loadModuleAnnotations('Imi\\Config', Object(Closure), '/^(?!((\\/usr\\/s...')
#8 /usr/src/app/vendor/imiphp/imi/src/Bean/Annotation.php(139): Imi\Bean\AnnotationLoader->loadModuleAnnotations('Imi\\Config', Object(Closure), '/^(?!((\\/usr\\/s...')
#9 /usr/src/app/vendor/imiphp/imi/src/Bean/Annotation.php(64): Imi\Bean\Annotation->loadModuleAnnotations('Imi', false)
#10 /usr/src/app/vendor/imiphp/imi/src/Bean/Scanner.php(37): Imi\Bean\Annotation->initByNamespace('Imi')
#11 /usr/src/app/vendor/imiphp/imi-workerman/src/WorkermanApp.php(109): Imi\Bean\Scanner::scanImi()
#12 /usr/src/app/vendor/imiphp/imi/src/App.php(87): Imi\Workerman\WorkermanApp->loadRuntime()
#13 /usr/src/app/vendor/imiphp/imi-workerman/bootstrap.php(64): Imi\App::run('ImiApp', 'Imi\\Workerman\\W...')
#14 /usr/src/app/vendor/imiphp/imi-workerman/bin/imi-workerman(3): Imi\Workerman\{closure}()
#15 {main}
  thrown in /usr/src/app/vendor/imiphp/imi/src/Bean/AnnotationParser.php on line 786
```

I found that previous version (**2.0.11**) worked.

Regards,

**Comment by Yurunsoft** [comment]
Thank you. I'll solve this problem today.

**Comment by Yurunsoft** [comment]
The problem has been solved in v2.0.13. @waghanza

**Comment by waghanza** [comment]
Working in https://github.com/the-benchmarker/web-frameworks/pull/3958. Thanks
